aea/cli/run.py

```diff
@@ -258,13 +259,27 @@ def _setup_connection(
     default=False,
     help="Install all the dependencies before running the agent.",
 )
+@click.option(
+    "--skip-consistency-check",
+    "skip_check",
+    is_flag=True,
+    required=False,
+    default=False,
+    help="Skip consistency check..",
+)
```

Comment:
optionally. for backward-compatibility, `aea run` can skip the checks introduced in this PR by using the option `--skip-consistency-check`